modules/gapi/src/compiler/gislandmodel.cpp

```diff
@@ -270,6 +286,9 @@ void GIslandModel::compileIslands(Graph &g, const ade::Graph &orig_g, const GCom
             auto island_exe = island_obj->backend().priv()
                 .compile(orig_g, args, topo_sorted_list);
             GAPI_Assert(nullptr != island_exe);
+
+            island_exe->compile(ins_data, outs_data);
```

Comment:
I'd remove this method from the base class for sure, let's bass these `ins_data`/`outs_data` in the above backend's `compile()`

Reply:
Fixed